### PR TITLE
Fix quickstart tutorial when running via `pip  install learnosity-sdk[quickstart]`

### PR DESCRIPTION
**Pull request by ferdia-sopermaccafraidh-lrn**
This resolves the issue when running the quickstart guide when installinga as a library from the documentation.

```
pip install learnosity-sdk[quickstart]
```

Working now shown below locally in a fresh venv and install from the locally created wheels.

image

## Checklist

- [ ] Feature
- [x] Bug
- [ ] Security
- [ ] Documentation

- [ ] ChangeLog.md updated

- [ ] Tests added
- [ ] All testsuites passed
- [ ] `make dist` completed successfully

### CODE REVIEW

**Comment by ferdia-sopermaccafraidh-lrn** [comment]
@walsh-conor this is my first PR after joining the company, so I could be missing something. If you have a moment to review I would appreciate it! 😄 

Please review with caution though due to my new status! 😅

File: ChangeLog.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v0.3.9] - 2024-08-15
```
Comment (shtrom) [inline ChangeLog.md:10]: The version stamping should be done at release time https://github.com/Learnosity/learnosity-sdk-python/blob/master/CONTRIBUTING.md#deploying-to-pypi

The ChangeLog will be automatically updated to include all `Unreleased` changes.

Comment (ferdia-sopermaccafraidh-lrn) [reply ChangeLog.md:10]: Thanks @shtrom for the advice! I'll revert those changes now 👍

File: learnosity_sdk/_version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = 'v0.3.8'
+__version__ = 'v0.3.9'
```
[the remark this answers is not in the document]
Comment (ferdia-sopermaccafraidh-lrn) [reply learnosity_sdk/_version.py]: Removed now 👍

File: ChangeLog.md (hunk 1 above)
Comment (ferdia-sopermaccafraidh-lrn) [reply ChangeLog.md]: I've updated the changelog now and added this under the `Unreleased` section.

@shtrom could you check it conforms with the usual standards?

**Review by shtrom — APPROVED** [review]
LGTM!

**Pull request merged**